README.md

```diff
@@ -76,7 +97,9 @@ You have now 3 JSON tables, let's query these tables using SQL with Apache Drill
 
 In this section you will learn how to use DB shell to query JSON table but also how to update a document with a new field.
 
-As mapr user in a terminal enter the following commands to get familiar with the shell
+As mapr user in a terminal enter the following commands to get familiar with the shell.
+
+Even when running MapR as sandbox container, one could use DB shell from the host machine.
```

Comment:
Change to:

If you are running MapR as a sandbox container, you can still use DB shell from the host machine.

Reply:
Right.